gdzig_bindings/builtin/variant.zig

```diff
@@ -136,7 +136,7 @@ pub const Variant = extern struct {
                 RID => .rid,
                 Signal => .signal,
                 String => .string,
-                StringName => .string_name,
+                []const u8, []u8, StringName => .string_name,
```

Comment:
this is incorrect, since string slices need to be converted to a String/StringName before creating a Variant. was this necessary for something, or is this just leftover from debugging?